src/pyflex/window_selector.py

```diff
@@ -184,6 +184,21 @@ def default(self, obj):
             except TypeError:
                 filename.write(j.encode())
 
+    def write_text(self, filename):
```

Comment:
I think the purpose is for those people who wants to use the window output directly for other programs other than python. The plain txt file contains much less information than the json file, but it is more friendly for other programs like C and FORTRAN to read in.

I will add the text :)